service/src/test/java/uk/gov/hmcts/reform/fpl/utils/MaskHelperTest.java

```diff
@@ -42,8 +41,8 @@ void shouldMaskEmailMultipleTimes() {
 
     @Test
     void shouldNotMaskEmailInEmptyString() {
-        assertThat(maskEmail(null, "test@test.com")).isEqualTo(null);
-        assertThat(maskEmail("", "test@test.com")).isEqualTo("");
+        assertThat(maskEmail(null, "test@test.com")).isNull();
+        assertThat(maskEmail("", "test@test.com")).isNull();
```

Comment:
Should this last one not be `isEmpty()`?

Reply:
of course, fiexed